internal_transfer_with_agreed_amount/wizards/internal_transfer_multicurrency.py

```diff
@@ -7,7 +7,10 @@ class InternalTransferMulticurrency(models.TransientModel):
 
     agreed_amount = fields.Monetary(help="Agreed amount in the currency of the destination bank.")
     currency_id = fields.Many2one(
-        "res.currency", string="Currency", required=True, default=lambda self: self._get_currency()
+        "res.currency",
+        string="Currency",
```

Comment:
Redundant string attribute, it's the same as the auto-generated one

Reply:
Removed string attribute.